lib/rex/text/wrapped_table.rb

```diff
@@ -70,8 +71,7 @@ def initialize(opts = {})
     # updated below if we got a "Rows" option
     self.rows     = []
 
-    # TODO: Discuss a cleaner way to handle this information
-    self.width    = opts['Width']   || ::IO.console.winsize[1]
+    self.width    = opts['Width']   || ::IO.console&.winsize&.[](1) || ::BigDecimal::INFINITY
```

Comment:
Safe navigation of arrays has quirky syntax IMO. I believe if console is defined, winsize should be a valid array:

```suggestion
    self.width    = opts['Width']   || ::IO.console&.winsize[1] || ::BigDecimal::INFINITY
```

However - I'll err on the side of caution though, in case there's cross-platform differences.

Reply:
Checking the code it doesn't seem like this scenario should arise, but i'm not strongly opinionated

https://github.com/ruby/ruby/blob/83e62d77981adeb2968a49c3176a7a940c8ee29e/ext/io/console/console.c#L780-L799